src/net_processing.cpp

```diff
@@ -2421,9 +2415,22 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
             // Get recent addresses
             m_connman.PushMessage(&pfrom, CNetMsgMaker(greatest_common_version).Make(NetMsgType::GETADDR));
             pfrom.fGetAddr = true;
+        }
 
-            // Moves address from New to Tried table in Addrman, resolves
-            // tried-table collisions, etc.
+        if (!pfrom.IsInboundConn()) {
+            // For non-inbound connections, we also update the addrman to
```

Comment:
Remove the word 'also'. Comments have a habit of migrating around in the codebase, and joining words like also lose their meaning if additional code/comments are added around the existing comments.

Reply:
Done.